ci-automation/ci_automation_common.sh

```diff
@@ -61,7 +61,16 @@ function update_and_push_version() {
 
     git commit --allow-empty -m "New version: ${version}"
 
-    git tag -f "${version}"
+    TAG_ARGS=()
```

Comment:
Let's make this variable local to the function:

```suggestion
    local -a TAG_ARGS
```